resources/definitions/os_discovery/xos.yaml

```diff
@@ -17,3 +17,36 @@ modules:
                 num_oid: '.1.3.6.1.4.1.1916.1.32.1.2.{{ $index }}'
                 index: 0
                 type: extreme-cpu
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - EXTREME-SYSTEM-MIB::extremeFanNumber
```

Comment:
This is an unnecessary snmpwalk.

According to the MIB, the index of extremeFanStatusTable is extremeFanNumber.